FAQ.md

```diff
@@ -255,6 +256,13 @@ If you just ran `dep init`, there should be a copy of your original vendor direc
 > dep assumes complete control of vendor/, and may indeed blow things away if it feels like it.
 -[@peterbourgon in #206](https://github.com/golang/dep/issues/206#issuecomment-277139419)
 
+## How do I make `dep` resolve dependencies from my `GOPATH`?
+By default, `dep` runs in network mode and would download all the dependencies
+from the network while initializing (`init`) and restoring (`ensure`). `init`
+provides an option to scan the `GOPATH` for dependencies by doing
+`dep init -gopath`, which falls back to network mode when the packages are not
```

Comment:
@sdboyer You may be getting a bit too hung up on what dep really does vs. what people are trying to do. 

This is me paraphrasing what I was asked a few times at GopherCon:
> My code works based on what's in my GOPATH, but otherwise I haven't been doing any dependency management or vendoring. I want to use dep, and have other people be able to use my project too, and have everything "just work" like it does on my machine.

The FAQ just needs to convey that dep don't use your GOPATH unless you tell it to, and if it can't get things to work with what's in the GOPATH, it will use the information from your dependencies' repositories to fill in the gaps.